src/site/markdown/index.md

```diff
@@ -512,7 +512,9 @@ This module provides some extension to JSON-B.
 
 #### Polymorphism
 
-This extension provides a way to handle polymorphism:
+This extension shouldn't be used anymore if you don't absolutely rely on the JSON format it generates/parses.
```

Comment:
What about "If your aim is only to be polymorphic you can use the built-in JSON-B feature but some cases remain unhandled and this extension enable them (mainly JSON-B to JSON-B cases like configurations and service to service calls but unlikely javascript to java cases)".